juju/juju
# CAAS workers use new application config

## Description of change

The CAAS workers use the new application config infrastructure to create service and ingress resources.

## QA steps

deploy kubernetes-core bundle and make a model
note the IP address of the worker machine
deploy my gitlab charm using the above IP as a config attribute:
  juju deploy ./gitlab --config juju-external-hostname=10.152.144.191.xip.io
expose the app
  juju expose gitlab

check that the app can be reached via a browser
